tlspuffin/Cargo.toml

```diff
@@ -258,6 +260,8 @@ wolfssl = { path = "../crates/wolfssl", optional = true }
 wolfssl-sys = { path = "../crates/wolfssl-sys", features = [], optional = true }
 foreign-types = { version = "0.5.0", optional = true }
 
+tls-harness = { path = "../puts/harness", package="tlspuffin-harness-sys", features = ["tls"], optional = true }
```

Comment:
Note to myself: Is this generic enough to support e.g. SSH? Why is there "tls" in features?

Reply:
The `tlspuffin-harness-sys` crate will handle the creation of PUTs regardless of protocols. The `tls` feature is here to limit the PUTs that will be linked into `tlspuffin`: We only want the PUTs that implement the TLS protocol. Respectively, in the future, `sshpuffin` will depend on this crate with the `ssh` feature.

Note that we plan to support more fine grain control over which PUTs are included, through environment variables:

- include all PUTs for  vendor libs in $VENDOR_DIR
```sh
    $ [VENDOR_DIR=...] cargo build -p tlspuffin
```

- filter PUTs included in build, by name:
```sh
    $ [VENDOR_DIR=...] [WITH_PUT=openssl111k,wolfssl430-asan,...] cargo build -p tlspuffin
```

Unfortunately, as cargo don't support more complex features, this is the only way I have found to provide this level of granularity, while providing sane defaults.

Reply:
> Note to myself: Is this generic enough to support e.g. SSH? Why is there "tls" in features?

Beware: this is the Cargo.toml in `tlspuffin` so it is dedicated to tls. Currently, the script and build arch. is `ProtocolBehavior` specific but we could eventually factor out the parts that are not specific in a dedicated crate.